cmake/third_party/azure_sdk.cmake

```diff
@@ -0,0 +1,81 @@
+include(GNUInstallDirs)
+include(${CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
+cmake_minimum_required(VERSION 3.13)
+
+# Skip the AZURE SDK build step if the extension is not enabled.
+if(NOT ENABLE_AZURE)
```

Comment:
Is it reasonable to do this in the `CMakeLists.txt` that includes this directory? That way if you have an older cmake and you're not building this extension, it won't fail just because we need to run this file and `return()`.

Reply:
i've moved it up a level so if the flag is not set we dont include `azure_sdk.cmake`

Reply:
Thanks!